### PR TITLE
Add links to technology behind Wasabi

### PR DESCRIPTION
**Pull request by RiccardoMasutti**

### CODE REVIEW

File: docs/why-wasabi/README.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ The motivation behind Wasabi is to provide the *strongest privacy for its users*
...
-- It has *advanced coin control features* to avoid linking coins from different transactions.
-- It's the only *Light wallet that doesn't fail against network analysis*.
-- *Tor anonimity network is integrated into the wallet*, making it harder to trace your transaction's path.
-- *It implements Chaumian CoinJoin* to allow unlinking coins from their past.
-- It complies with *Zero Link: The Bitcoin Fungibility Framework*.
+- It has [*advanced coin control features](/FAQ/FAQ-UseWasabi.html#coin-control-best-practices)* to avoid linking coins from different transactions.
+- It's the only *Light wallet that doesn't fail against [network analysis](/using-wasabi/NetworkLevelPrivacy.html)*.
+- *Tor anonimity network is integrated into the wallet*, [making it harder to trace your transaction's path](/FAQ/FAQ-GeneralBitcoinPrivacy.html#how-does-tor-protect-my-network-level-privacy).
+- *It implements [Chaumian CoinJoin](https://github.com/nopara73/ZeroLink/#ii-chaumian-coinjoin)* to allow unlinking coins from their past.
+- It complies with *[Zero Link: The Bitcoin Fungibility Framework](https://github.com/nopara73/ZeroLink/)*.
```
Comment (yahiheb) [inline docs/why-wasabi/README.md]:
```suggestion
- It complies with [Zero Link: The Bitcoin Fungibility Framework](https://github.com/nopara73/ZeroLink/).
```

**Comment by yahiheb** [comment]
No, it is to make it italic.
If this is accepted then there is no need to make it italic because it will be clickable and have a different color.

**Comment by RiccardoMasutti** [comment]
> No, it is to make it italic.
> If this is accepted then there is no need to make it italic because it will be clickable and have a different color.

I left it because it was present before my commit

**Comment by yahiheb** [comment]
> > No, it is to make it italic.
> > If this is accepted then there is no need to make it italic because it will be clickable and have a different color.
> 
> I left it because it was present before my commit

If this is accepted then there is no need to make it italic because it will be clickable and have a different color.

**Comment by yahiheb** [comment]
Captureq

**Comment by RiccardoMasutti** [comment]
Should be ready @MaxHillebrand

**Pull request merged**